package.json

```diff
@@ -27,7 +27,7 @@
     "localforage": "1.5.3",
     "localforage-observable": "1.4.0",
     "marked": "mattermost/marked#802e981ade71149a497cbe79d12b8a3f82f7657e",
-    "mattermost-redux": "mattermost/mattermost-redux#917fbf0969ff6a042d536cb711063396decfb4b3",
+    "mattermost-redux": "mattermost/mattermost-redux",
```

Comment:
Why not set commit to `58e27c126e9a0ef4278001df37933436a08517c5`? I think this will cause change to `yarn.lock` every time.

Reply:
It'll only change if you run `yarn upgrade`, and it allows you to actually run `yarn upgrade mattermost-redux` to get the latest version. I'm going off what Elias has us using in the mobile app